pkg/kindsys/report.json

```diff
@@ -1,907 +1,580 @@
+      "name": "zipkin_query",
+      "type": "composable",
+      "maturity": "planned"
     }
-  ],
-  "raw": [
-    {
-      "name": "SVG",
-      "pluralName": "SVGs",
-      "machineName": "svg",
-      "pluralMachineName": "svgs",
-      "lineageIsGroup": false,
-      "maturity": "merged",
-      "extensions": [
-        "svg"
-      ]
+  },
+  "dimensions": {
+    "maturity": {
+      "experimental": {
+        "name": "experimental",
+        "items": [],
+        "count": 0
+      },
+      "mature": {
+        "name": "mature",
+        "items": [],
+        "count": 0
+      },
+      "merged": {
+        "name": "merged",
+        "items": [
+          "annolist_panel",
+          "barchart_panel",
+          "bargauge_panel",
+          "dashboard",
+          "dashlist_panel",
+          "gauge_panel",
+          "histogram_panel",
+          "news_panel",
+          "piechart_panel",
+          "playlist",
+          "stat_panel",
+          "svg",
+          "team",
+          "text_panel"
+        ],
+        "count": 14
+      },
+      "planned": {
+        "name": "planned",
+        "items": [
+          "alertgroups_panel",
+          "alertlist_panel",
+          "alertmanager_dsoptions",
+          "alertmanager_query",
+          "apikey",
+          "cloudwatch_dsoptions",
+          "cloudwatch_query",
+          "dashboard_dsoptions",
+          "dashboard_query",
+          "datasource",
+          "debug_panel",
+          "elasticsearch_dsoptions",
+          "elasticsearch_query",
+          "flamegraph_panel",
+          "folder",
+          "geomap_panel",
+          "gettingstarted_panel",
+          "grafana_azure_monitor_datasource_dsoptions",
+          "grafana_azure_monitor_datasource_query",
+          "grafana_dsoptions",
+          "grafana_query",
+          "graph_panel",
+          "graphite_dsoptions",
+          "graphite_query",
+          "icon_panel",
+          "jaeger_dsoptions",
+          "jaeger_query",
+          "live_panel",
+          "logs_panel",
+          "loki_dsoptions",
+          "loki_query",
+          "mssql_dsoptions",
+          "mssql_query",
+          "mysql_dsoptions",
+          "mysql_query",
+          "nodegraph_panel",
+          "parca_dsoptions",
+          "parca_query",
+          "phlare_dsoptions",
+          "phlare_query",
+          "postgres_dsoptions",
+          "postgres_query",
+          "prometheus_dsoptions",
+          "prometheus_query",
+          "query",
+          "queryhistory",
+          "serviceaccount",
+          "stackdriver_dsoptions",
+          "stackdriver_query",
+          "table_old_panel",
+          "tempo_dsoptions",
+          "tempo_query",
+          "testdata_dsoptions",
+          "testdata_query",
+          "thumb",
+          "traces_panel",
+          "user",
+          "welcome_panel",
+          "xychart_panel",
+          "zipkin_dsoptions",
+          "zipkin_query"
+        ],
+        "count": 61
```

Comment:
more curious than anything, but is having a count that's an aggregate across the kind categories a problem for viz?